templates/zerver/invite_user.html

```diff
@@ -15,15 +15,17 @@ <h3 id="invite-user-label">{% trans %}Invite users to Zulip{% endtrans %}</h3>
                         <textarea rows="2" id="invitee_emails" name="invitee_emails" placeholder="{{ _('One or more email addresses...') }}"></textarea>
                     </div>
                     {% if is_admin %}
-                    <label class="control-label" id="invite_as_admin" for"invite_as_admin">{{ _('User(s) join as') }}</label>
                     <div class="controls">
-                        <label class="radio">
-                            <input type="radio" name="invite_as_admin" value="false" checked/>{{ _('Regular users') }}
-                        </label>
-                        <label class="radio">
-                            <input type="radio" name="invite_as_admin" value="true"/>{{ _('Organization administrators') }}
+                        <label class="checkbox display-block">
+                            <input type="checkbox" name="invite_as_admin" value="true" checked="checked" />
```

Comment:
The default here should be false.
Also, I don't know too much about frontend development, but I'm surprised this doesn't require other changes. Do checkboxes send the same "true" and "false" to the backend that the radio button was sending?

Reply:
Earlier I thought it would return null but I checked and it works perfectly fine so I commited the changes.

Reply:
A few comments:
* the default here should be false
* could you check to see why this is working / where the conversion from this value to the backend true/false is being made? 
* I just realized it would be great to add a casper test for this, since it would be really bad if this breaks in the future, and non-admins are accidentally invited as admins 

Thanks!

Reply:
@rishig Related backend is at : zerver/views/invite.py
Line 22,27 where default is false and true is for administrator users .

Reply:
yup. What I mean is that on the frontend the default should also be false (i.e. the box should be unchecked by default)

Reply:
so I guess removing : checked="checked" part from the input tag will do it

Reply:
that would be my guess, and I would test manually after that